models/issue_reaction.go

```diff
@@ -112,13 +136,13 @@ func CreateReaction(opts *ReactionOptions) (reaction *Reaction, err error) {
 
 	reaction, err = createReaction(sess, opts)
 	if err != nil {
-		return nil, err
+		return
```

Comment:
I would prefer it the way it was and removing return variable names from func definition

Reply:
done